src/components/utils/onfidoApi.js

```diff
@@ -2,12 +2,14 @@ import { performHttpReq } from '../utils/http'
 import Tracker from '../../Tracker'
 import forEach from 'object-loops/for-each'
 import { humanizeField } from '../utils'
+import { isOfFileType } from '../utils/file.js'
 
 const formatError = ({response, status}, onError) =>
   onError({status, response: JSON.parse(response)})
 
 export const uploadDocument = (data, token, onSuccess, onError) => {
-  const validations = {'detect_document': 'error', 'detect_glare': 'warn'}
+  const glareValidation = isOfFileType(['pdf'], data.file) ? false : 'warn'
```

Comment:
I would be tempted with not adding at all the key `detect_glare`. Simply because I'm not sure how strong and reliable the contract is for this case.